pom.xml

```diff
@@ -3,16 +3,19 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.7</version><!-- which version of Jenkins is this plugin built against? -->
+    <version>3.10</version><!-- which version of Jenkins is this plugin built against? -->
```

Comment:
Is there a Jenkins version 3.10? Or is this some other version?

Reply:
This is just jenkins-plugin-support version it doesn't map to jenkins core version directly.

This is required for incremental to work

Reply:
Actually lets drop the misleading comment